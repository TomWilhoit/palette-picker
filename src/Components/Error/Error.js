import React from 'react';

export const Error = (props) => <h2 className="error-display">{props.message}</h2>