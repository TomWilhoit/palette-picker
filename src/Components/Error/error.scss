.error-display {
  @include top-right-message;
}