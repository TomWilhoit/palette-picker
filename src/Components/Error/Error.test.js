import React from "react";
import { Error } from "./Error";
import { shallow } from "enzyme";

describe("Error", () => {
  let wrapper;
  let mockErrMessage = "Hello"
  beforeEach(() => {
    wrapper = shallow(<Error message={mockErrMessage} />);
  })

  it("should match the snapshot with all data passed in", () => {
    expect(wrapper).toMatchSnapshot();
  })
})