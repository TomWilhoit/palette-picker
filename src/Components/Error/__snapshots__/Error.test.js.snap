// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot with all data passed in 1`] = `
<h2
  className="error-display"
>
  Hello
</h2>
`;
